tests/monotouch-test/MediaToolbox/FormatNamesTest.cs

```diff
@@ -25,9 +27,14 @@ public class FormatNamesTest {
 		[Culture ("en")]
 		public void LocalizedNameForMediaType ()
 		{
+#if MONOMAC
+			if (!TestRuntime.CheckMacSystemVersion (10, 11))
+				Assert.Ignore ("Requires macOS 10.11");
+#else
```

Comment:
Same: replace `CheckSystemVersion` with call to `TestRuntime.CheckXcodeVersion`.